cmd/configure.go

```diff
@@ -0,0 +1,108 @@
+package cmd
```

Comment:
Keep this command in `cmd/configure/configure.go`

Reply:
kept it